### ISSUE TITLE
[Matsuribayashi] Wrong background textbox in matsuribayashi in OG mode

### ISSUE THREAD

**Issue by drojf** [07th-mod/higurashi-assembly#70]
When playing in OG mode, it's supposed to use a textbox that fills the entire screen.

In matsuribayashi, when you change to OG mode, it will use the correct textbox. However, it will reset to the 'NVL' mode textbox the next time you load the game (or just randomly some time after you enable OG mode).

I'm not sure if this is related to #69 since this seems to only occur in matsuribayashi

**Comment by drojf** [comment]
I trialed a fix for this issue, and it seemingly also fixes #68 and #69 somehow (even though the changes only fix this issue, #70). I guess if the textbox background is not set correctly it causes lots of weird stuff to happen...

Around the end of last year I added OG mode. There are two places in the code which set the background textbox, and in one place it was very different for Matsuribayashi as compared to the other chapters. I must have made a mistake or overlooked when merging into Matsuribayashi, since the code to set the textbox for OG mode was not there.

Specifically, it is the `ShowLayerBackground` function which differs:

branch mod
https://github.com/07th-mod/higurashi-assembly/blob/3d821bc67b29e86f844ceb461a95bdf48e9723fd/Assets.Scripts.UI/MainUIController.cs#L118-L187

branch matsuri-mod
https://github.com/07th-mod/higurashi-assembly/blob/799941d46d5d9df65c927a6101d12b9ecc1b0208/Assets.Scripts.UI/MainUIController.cs#L118-L159
